packages/api/src/routes/settings.ts

```diff
@@ -126,7 +126,7 @@ const updateSettingsSchema = z
 router.post(
   "/",
   asyncHandler(async (req: Request, res: Response) => {
-    const id = getCxIdOrFail(req);
+    const cxId = getCxIdOrFail(req);
```

Comment:
updated this since was confusing. One of the only places cxId is named id insted of cxId